### PR TITLE
Include edit command in user guide

### PR DESCRIPTION
**Pull request by CharlesLee01**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -79,6 +79,29 @@ Examples:
...
+
+Edits a dog/owner/program from Pawbook.
+name, Set<Session> sessionSet, Set<Tag> tags
+Format:
+```
+edit dog d/<DOG ID> n/<NAME> b/<BREED> d/<DATEOFBIRTH> s/<SEX> o/<OWNERID> [t/TAGS]...
+edit owner o/<OWNER ID> n/<NAME> p/<PHONE> e/<EMAIL> a/<ADDRESS> [t/TAGS]...
+edit program o/<PROGRAM ID> n/<NAME> [s/SESSION]... [t/TAGS]...
+```
+
```
Comment (kouyk) [inline docs/UserGuide.md:92]: can there be examples here as well?

Comment (CharlesLee01) [reply docs/UserGuide.md:92]: Sure!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -79,6 +79,29 @@ Examples:
 3. Deletes the program with ID 3 in Pawbook.<br>
    Command: `delete program p/3`
 
+### Edit Command
+
+Edits a dog/owner/program from Pawbook.
+name, Set<Session> sessionSet, Set<Tag> tags
```
Comment (kouyk) [inline docs/UserGuide.md:85]: this line is copy pasted from somewhere?

Comment (CharlesLee01) [reply docs/UserGuide.md:85]: I just wanted to make it easier for me to reference the attributes of a program when writing the DG. Forgot to remove it, thx!

**Pull request closed without merging**